powertools-validation/src/test/resources/sqs.json

```diff
@@ -1,5 +1,5 @@
 {
-  "records": [
+  "Records": [
```

Comment:
Why this change ? 🤔

Reply:
Records with big R is the correct one. But before the availability of the [aws-lambda-java-serialization](https://github.com/aws/aws-lambda-java-libs/tree/master/aws-lambda-java-serialization) library, we could not deserialize an SQS event properly (Mixins needed). This is why there was a small r. Now, I'm using the library in the test, so I could correct the json file.